src/Microsoft.AspNetCore.SignalR.Client/HubConnection.cs

```diff
@@ -203,7 +204,7 @@ private void OnDataReceived(byte[] data, MessageType messageType)
                         _logger.LogWarning("Dropped unsolicited Stream Item message for invocation '{invocationId}'", streamItem.InvocationId);
                         return;
                     }
-                    DispatchInvocationStreamItem(streamItem, irq);
+                    DispatchInvocationStreamItemAsync(streamItem, irq);
```

Comment:
If we want to consider ordering of streamed items, that's a whole new design concept as the frames could get reordered in transit (so it doesn't even matter how we queue on the client). It's not necessarily required, as the user can stream items tagged with an order and do buffering.